### PR TITLE
WIP: home / base css cleanup

### PR DESCRIPTION
**Pull request by jameslzhu**
Do not merge. Work in progress rewriting CSS with http://tachyons.io/, in particular to support mobile better.

### CODE REVIEW

File: hknweb/templates/landing/home.html [hunk 1]
```diff
@@ -1,70 +1,32 @@
...
     </div>
 
-    <div class="tutoring-column">
+    <div class="db tutoring-column center measure">
         <div class="tutoring-panel">
             <div class="tutoring-title">
                 Today's Tutoring Hours
             </div>
-            <span class="tutoring-date"><div>April 22, 2019</div></span>
+            <span class="tutoring-date">{% now "DATE_FORMAT" %}</span>
```
Comment (jameslzhu) [inline hknweb/templates/landing/home.html]: Should we pull this out into a separate pull request?

**Comment by ochan1** [comment]
Closing because abandoned WIP
The future plan is to use Node and React for mobile and styling
